.vscode/settings.json

```diff
@@ -48,10 +48,12 @@
         "NODEFOO",
         "onsi",
         "resmgr",
+        "resmgrsockets",
         "resourcemanagerv",
         "sapcloud",
         "testbroken",
         "unmarshalled",
         "unmarshalling"
-    ]
+    ],
+    "cSpell.enabled": true
 }
```

Comment:
Missing new line at the end of the file.

Reply:
The is auto generated file - and its fine without it ;)